README.md

```diff
@@ -34,13 +34,6 @@ It integrates the following tools:
 
 You can see the template in action in the [example project](https://github.com/DiamondLightSource/python-copier-template-example). This is an up to date expansion of the template to illustrate how it looks with all the options enabled.
 
-## Create a new project via Developer Portal
```

Comment:
The URL below does actually work to create repos from the dev portal (at least for me), but I was having a discussion with @gilesknap as to whether we wanted to teach this as we then have to teach a different mechanism for repo creation to repo updating. I'm on the fench, could you give it a go and see what you think please?

Reply:
![image](https://github.com/user-attachments/assets/a925f320-d51b-4408-b85d-feeb29b2416b)

I was able to click through and select the options from the template, but it didn't seem to give me defaults, and after clicking review I got this failure page. If it could make repos on the DLS org I think there's value add, but losing the defaults that depend on the previous answers makes it more frustrating